spec/lib/publishing_api_finder_publisher_spec.rb

```diff
@@ -19,7 +19,7 @@
       }
     end
 
-    def make_metadata base_path, overrides = {}
+    def make_metadata(base_path, overrides = {})
```

Comment:
The change on this line is excellent, but unrelated to the rest of the commit... can we make it a separate one?